trailbot_main/launch/trailbot_demo1.launch.py

```diff
@@ -0,0 +1,212 @@
+import os
```

Comment:
Is the goal for this to be the main launch file? Or should the fsm only be a part of this demo?